packages/fuels-test-helpers/src/lib.rs

```diff
@@ -157,22 +162,20 @@ pub async fn setup_test_client(
     let mut config = node_config.unwrap_or_else(Config::local_node);
     config.chain_conf = chain_conf;
 
-    let srv = FuelService::new_node(config).await.unwrap();
+    let srv = FuelService::new_node(config)
+        .await
+        .map_err(|err| fuels_core::types::errors::Error::InfrastructureError(err.to_string()))?;
```

Comment:
you can import `fuels_core::types::errors::{error, Error}` and do 
```suggestion
        .map_err(|err| error!(InfrastructureError, "{err}"))?;
```

Reply:
I saw that this was the practice and tried to do it like this, but I ran into problems. Maybe because of the flags we have. I'll try again